script/setup/install-critools

```diff
@@ -21,7 +21,7 @@
 set -eu -o pipefail
 
 go get -u github.com/onsi/ginkgo/ginkgo
-CRITEST_COMMIT=v1.18.0
+CRITEST_COMMIT=89384cc13a27bb9128553c9fe75a7cc07c6a95bb
```

Comment:
Do we know if a release is pending for this? https://github.com/kubernetes-sigs/cri-tools/compare/v1.18.0...89384cc13a27bb9128553c9fe75a7cc07c6a95bb

(there's one more commit in master)

Reply:
cri-tools is fighting through problems in CI caused by dockershim changes in k/k.. (namely a an experimental flag that cri-tools used to run dockershim as a process in kublet was removed). Till a mitigation is provided or the dockershim test is removed, the version of kubernetes that cri-tools runs against is frozen. @dims has a mitigation involving running dockershim as a separate process but we don't have any owners for dockershim yet. @thaJeztah you wouldn't know anyone that would want to host dockershim would you?

Reply:
Gotcha! I heard some bits about the dockershim; overall the direction for users should be to use containerd (as it was designed to provide a runtime for kubernetes), but I realise there's still a fair share of users to use dockershim.

I must admit I'm horrible at k8s (only so much time in a day), but I can ask around if there's intent to look after it.